drizzlepac/haputils/analyze.py

```diff
@@ -364,6 +364,8 @@ def analyze_data(input_file_list, log_level=logutil.logging.DEBUG, type=""):
     for i, input_file in enumerate(input_file_list):
         header_hdu = 0
         header_data = getheader(input_file, header_hdu)
+        science_hdu = 1
```

Comment:
FLT or FLC images can have multiple SCI extensions (representing different chips) depending upon the detector and how the data was acquired.  You need to determine how many SCI extensions are in the file, and then check all of the extensions.  There would only be 1 or 2 SCI extensions. 

If for some strange observation with two SCI extensions, only one extension contains all zeros, the processing should continue.  A message should be issued indicating "Warning, (SCI, n) is all zeros, but processing will continue." or something like this.  Only if all the SCI extensions are zero  in the file should the processing be set to FALSE.